birdhouse/config/flyingpigeon/docker-compose-extra.yml

```diff
@@ -0,0 +1,24 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  flyingpigeon:
+    image: birdhouse/flyingpigeon:1.6
+    container_name: flyingpigeon
+    environment:
+      - PYWPS_CFG=/wps.cfg
+    ports:
+      - "8093:8093"
+    volumes:
+      - ./config/flyingpigeon/wps.cfg:/wps.cfg
+      - /tmp
+    depends_on:
+      - postgres
```

Comment:
This should technically be in a `postgres` fragment but since `./config/postgres` is specified in `COMPONENT_DEPENDENCIES`, I think we can keep this here.  Not going to create a compose fragment file just for 2 lines.